### DISCUSSION TITLE
is there a way to get redstone faster?

### DISCUSSION THREAD

**Discussion by hans2356** [Yasser444o/Skybedrock#10] [Q&A]
i am big fan of skybedrock since i started last week.

I am almost newbee on skyblock and I almost spended time for making redstone things in vanilla minecraft.

I made cobblestone generator but I realized some cobblestone burns out after they touch the lava.

I want to make efficient cobblestone generator at early stage, for that, I need to get redstone at early period.

is there any way to get redstones at early period without getting from cleric?

**Comment by Yasser444o** [comment]
Yes indeed, find the swamp hut and build a simple witch farm! it doesn't need pistons, hoppers or even tridents to be the most efficient, all you need to do is to find the spot where witches spawn, and make the witches fall to there death as soon as they spawn for quick and easy redstone. 
You can use the How to play screen to learn more about the swamp hut spawning mechanics and i recommend watching a tutorial by silentwhisperer to learn how to find the spawning spot

**Comment by hans2356** [comment]
Thank you for responding.
I made a monster trap and got a few redstones!
I think I need to find swamp hut for more redstones but it seems that structure locator does not work.

**Reply by Yasser444o** [reply]
oh, sorry about that, you need to download the map again, and replace the behavior packs folder of your world with the one you downloaded.
